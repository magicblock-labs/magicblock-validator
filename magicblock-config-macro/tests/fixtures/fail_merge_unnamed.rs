use magicblock_config_macro::Mergeable;

#[derive(Default, Mergeable)]
struct TestConfig(u64, String);

fn main() {}
