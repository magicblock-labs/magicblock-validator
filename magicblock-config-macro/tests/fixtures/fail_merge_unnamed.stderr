error: Merge can only be derived for structs with named fields
 --> tests/fixtures/fail_merge_unnamed.rs:4:18
  |
4 | struct TestConfig(u64, String);
  |                  ^^^^^^^^^^^^^
